src/schemas.py

```diff
@@ -10,3 +10,16 @@ class Project(BaseModel):
 
 class ProjectDetails(Project):
     project_id: uuid.UUID = Field(default_factory=uuid.uuid4)
+
+
+class User(BaseModel):
+    user_id: uuid.UUID = Field(default_factory=uuid.uuid4)
+    name: str
+    email: str
+    password: str
+
+
+class Document(BaseModel):
+    document_id: uuid.UUID = Field(default_factory=uuid.uuid4)
+    title: str
+    file_path: str
```

Comment:
add an appropriate data type to validate it, i think pydantic has url data type or something like that